src/components/map/toolbar/toolbarModeButtons.tsx

```diff
@@ -2,12 +2,41 @@ import React from 'react';
 import classnames from 'classnames';
 import EditMode from '~/enums/editMode';
 import toolbarStore from '~/stores/toolbarStore';
+import routeBuilder from '~/routing/routeBuilder';
+import subSites from '~/routing/subSites';
+import navigator from '~/routing/navigator';
 import { RadioButton } from '../../controls';
 import * as s from './toolbarModeButtons.scss';
 
 export default class ToolbarModeButtons extends React.Component {
-    private toggleSelectedMode = (option: EditMode) => () => {
-        toolbarStore.setEditMode(option);
+    constructor(props: any) {
```

Comment:
Don't use props: any. Create dummy prop interface if we don't set any props manually, that is the way we have done in other parts of the project